client/app/scripts/superdesk-workspace/workspace.js

```diff
@@ -102,12 +102,12 @@
          */
         function getActiveWorkspace() {
             return desks.fetchCurrentDeskId()
-                .then(getActiveWorkspaceId)
+                .then(getActiveWorkspaceId())
```

Comment:
Is this indeed correct? `then()` expects a callback that is invoked on success, and not a promise object itself (but a callback can return the latter).